Orange/widgets/widget.py

```diff
@@ -198,32 +197,48 @@ def createPixmapWidget(self, parent, iconName):
         gui.rubber(self.warning_bar)
         self.warning_bar.setVisible(False)
 
-        self.topWidgetPart = gui.widgetBox(self,
-                                           orientation="horizontal", margin=0)
-        self.leftWidgetPart = gui.widgetBox(self.topWidgetPart,
-                                            orientation="vertical", margin=0)
+        self.want_main_area = self.want_graph or self.want_main_area
+
+        class Splitter(QSplitter):
```

Comment:
Makes sense. But Qt doesn't handle it well in OWWidget's namescape (I suppose because this function is called in widget's constructor and `self` is not yet _fully_ available by then).